cmd/bootkube/start.go

```diff
@@ -21,19 +21,22 @@ var (
 	startOpts struct {
 		assetDir        string
 		podManifestPath string
+		strict          bool
 	}
 )
 
 func init() {
 	cmdRoot.AddCommand(cmdStart)
 	cmdStart.Flags().StringVar(&startOpts.assetDir, "asset-dir", "", "Path to the cluster asset directory. Expected layout genereted by the `bootkube render` command.")
 	cmdStart.Flags().StringVar(&startOpts.podManifestPath, "pod-manifest-path", "/etc/kubernetes/manifests", "The location where the kubelet is configured to look for static pod manifests.")
+	cmdStart.Flags().BoolVar(&startOpts.strict, "strict", false, "Strict mode will cause bootkube to exit early if any manifests in the asset directory cannot be started.")
```

Comment:
nit: s/cannot be started/cannot be created/

Reply:
good comment. tweaked.